### ISSUE TITLE
colour contrast fail - menu

### ISSUE THREAD

**Issue by Cheesycats** [NHSDigital/data-viz-community-of-practice#8]
Menu fails colour contrast standards 

![image](https://github.com/user-attachments/assets/779ada1f-1c34-4351-a5da-417d27b98a72)
